ofrak_core/ofrak/core/uefi.py

```diff
@@ -0,0 +1,53 @@
+import os
+import asyncio
+import logging
+import tempfile
+from dataclasses import dataclass
+from subprocess import CalledProcessError
+
+from ofrak.component.unpacker import Unpacker
+from ofrak.resource import Resource
+from ofrak.core.filesystem import File, Folder, FilesystemRoot, SpecialFileType
+
+
+from ofrak.model.component_model import ComponentExternalTool
+from ofrak.core.pe.model import Pe
+
+LOGGER = logging.getLogger(__name__)
+
+UEFIEXTRACT = ComponentExternalTool("uefiextract", "https://github.com/LongSoft/UEFITool", "--help")
```

Comment:
Have you looked to see if this is on `apt` and `brew`? If so, would be good to add the respective package name(s) for those package managers here.

Reply:
Not on either